v1/calorie_mapper_v1.ipynb

```diff
@@ -226,25 +228,53 @@
    "metadata": {},
    "source": [
     "### Subtask 4\n",
-    "* Text matching to find the first match between the label array and the calorie_counts dataset, identify the matching row"
+    "* Design a database that holds the calorie_counts datasets then find the best match between the label array and the database"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 29,
+   "execution_count": 93,
    "metadata": {},
    "outputs": [],
    "source": [
-    "# read the dataset, clean and arrange data into a dict for quicker lookup.\n",
-    "cal_data = pd.read_csv('../data/calorie_counts.csv').drop(['Serving'], axis=1)\n",
-    "cal_data['Food'] = cal_data['Food'].str.lower()\n",
-    "cal_dict = dict(zip(cal_data['Food'], cal_data['Calories']))\n",
+    "#create the connection to firebase\n",
+    "firebase_config = {'apiKey': \"AIzaSyDLBp2GpdSOd-p16l8ghvSKJ9X008WNAKk\",\n",
+    "  'authDomain': \"calorie-counts-2e344.firebaseapp.com\",\n",
+    "  'projectId': \"calorie-counts-2e344\",\n",
+    "  'storageBucket': \"calorie-counts-2e344.appspot.com\",\n",
+    "  'messagingSenderId': \"711145706388\",\n",
+    "  'appId': \"1:711145706388:web:1cd7d7d8010dc633b3b581\",\n",
+    "  'databaseURL': \"https://calorie-counts-2e344-default-rtdb.europe-west1.firebasedatabase.app/\"\n",
+    "}\n",
+    "firbase = pyrebase.initialize_app(firebase_config)\n",
+    "db = firbase.database()\n",
     "\n",
+    "\"\"\"\n",
+    "#convert the csv file to json format and upload it to firebase\n",
+    "with open('../data/calorie_counts.csv', newline='', encoding='utf-8') as csvfile:\n",
+    "    reader = csv.DictReader(csvfile, delimiter=',')\n",
+    "    calorie_data = {}\n",
+    "    list_of_special_chars =['.','’']\n",
+    "    for row in reader:\n",
+    "      for char in list_of_special_chars:\n",
+    "        if char in row['Food']:\n",
+    "          row['Food'] = row['Food'].replace(char,'')\n",
+    "      if '-' in row['Food']:\n",
+    "        row['Food'] = row['Food'].replace('-', ' ')\n",
+    "      if 'é' in row['Food']:\n",
+    "        row['Food'] = row['Food'].replace('é', 'e')\n",
```

Comment:
Nice, alternatively we could use regex or string concatenation like https://stackoverflow.com/a/29350747